forecast_script/docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: '2'
+services:
+  db:
+    image: kartoza/postgis:12.0
```

Comment:
Our stack is currently using the 11 image

Reply:
Ah ok